code/core/src/manager/components/sidebar/Search.stories.tsx

```diff
@@ -23,77 +22,101 @@ const getLastViewed = () =>
     .filter((item, i) => item.type === 'component' && item.parent && i % 20 === 0)
     .map((component) => ({ storyId: component.id, refId }));
 
+const setQueryParams = fn().mockName('setQueryParams');
+const getQueryParams = fn().mockName('setQueryParams');
+
 const meta = {
   component: Search,
   title: 'Sidebar/Search',
   parameters: { layout: 'fullscreen' },
+  args: {
+    dataset,
+    getLastViewed: (): Selection[] => [],
+    children: () => <SearchResults {...noResults} />,
+  },
+  render: ({ children, ...args }) => {
+    return <Search {...args}>{children}</Search>;
+  },
   globals: { sb_theme: 'side-by-side' },
   decorators: [
-    (storyFn: any) => (
-      <div style={{ padding: 20, maxWidth: '230px' }}>
-        <IconSymbols />
-        {storyFn()}
-      </div>
+    (storyFn) => (
+      <ManagerContext.Provider
+        value={
+          {
+            state: {},
+            api: {
+              emit: () => {},
+              on: () => {},
+              off: () => {},
+              getShortcutKeys: () => ({ search: ['control', 'shift', 's'] }),
+              selectStory: () => {},
+              setQueryParams,
+              getQueryParams,
+            },
+          } as any
+        }
+      >
+        <LayoutProvider>
+          <IconSymbols />
+          {storyFn()}
+        </LayoutProvider>
+      </ManagerContext.Provider>
     ),
   ],
 } satisfies Meta<typeof Search>;
+
 export default meta;
 
-const baseProps = {
-  dataset,
-  clearLastViewed: action('clear'),
-  getLastViewed: () => [] as Selection[],
+type Story = StoryObj<typeof meta>;
+
+export const Simple: Story = {};
+
+export const SimpleWithCreateButton: Story = {
+  args: {
+    showCreateStoryButton: true,
+  },
+};
+
+export const FilledIn: Story = {
+  args: {
+    initialQuery: 'Search query',
+  },
+};
+
+export const LastViewed: Story = {
+  args: {
+    getLastViewed,
+  },
```

Comment:
These lines have been removed, is that on purpose?

```suggestion
  args: {
    getLastViewed,
    children: ({ query, results, closeMenu, getMenuProps, getItemProps, highlightedIndex }) => (
      <SearchResults
        query={query}
        results={results}
        closeMenu={closeMenu}
        getMenuProps={getMenuProps}
        getItemProps={getItemProps}
        highlightedIndex={highlightedIndex}
      />
    )
  },
```

maybe this is just the equivalent of the default `    children: () => <SearchResults {...noResults} />,` arg?

Reply:
from what i remember, its because the default handles it. none of the others specify `children` other than the default